src/main.rs

```diff
@@ -338,15 +338,15 @@ fn inner_main() -> Result<ExitCode> {
 
         // Configure the file watcher.
         let (tx, rx) = channel();
-        let mut watcher = raw_watcher(tx)?;
+        let mut watcher = recommended_watcher(tx)?;
         for file in &cli.files {
             watcher.watch(file, RecursiveMode::Recursive)?;
         }
 
         loop {
             match rx.recv() {
                 Ok(e) => {
-                    if let Some(path) = e.path {
+                    for path in e?.paths {
```

Comment:
Maybe the list of paths should be passed into `run_once` instead? Or we should check `.py` in the list of paths and only run `run_once` once?

I'm not totally sure, the current state is the minimal changes to make it compile.